packages/sdk/src/evm/core/classes/contract-appuri.ts

```diff
@@ -45,12 +45,14 @@ export class ContractAppURI<TContract extends BaseContract>
    * ```
    * @twfeature AppURI
    */
-  public async get() {
-    if (detectContractFeature<AppURI>(this.contractWrapper, "AppURI")) {
+  public async get(): Promise<string> {
+    if (detectContractFeature<IAppURI>(this.contractWrapper, "AppURI")) {
       return await this.contractWrapper.readContract.appURI();
     }
 
-    return (await this.metadata.get()).appURI || "";
+    const appUri = (await this.metadata.get()).app_uri;
```

Comment:
This will throw an error if `app_uri` isn't on metadata - which can happen since metadata is untyped.